sbt-scala-native-crossproject/src/main/scala/scalanativecrossproject/NativePlatform.scala

```diff
@@ -0,0 +1,14 @@
+package scalanativecrossproject
+
+import sbt._
+import sbtcrossproject._
+import scalanative.sbtplugin._
+
+case object NativePlatform extends Platform {
+  def identifier: String = "native"
+  def sbtSuffix: String  = "Native"
+  def enable(project: Project): Project =
+    project.enablePlugins(ScalaNativePlugin)
+  val crossBinary: CrossVersion = ScalaNativeCrossVersion.binary
+  val crossFull: CrossVersion   = ScalaNativeCrossVersion.full
```

Comment:
Those two members should deprecated, like [the abstract definitions that they implement](https://github.com/portable-scala/sbt-crossproject/blob/b2cb6f8be82706183e79d34c8a5522365051594d/sbt-crossproject/src/main/scala/sbtcrossproject/Platform.scala#L10-L14).